2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_setup.py:_flush():76] Loading settings from /home/aneesh/.config/wandb/settings
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_setup.py:_flush():76] Loading settings from /home/aneesh/github/slot_vqa/wandb/settings
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'start_method': 'thread'}
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': '/home/aneesh/github/slot_vqa/train.py'}
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_init.py:_log_setup():428] Logging user logs to /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/logs/debug.log
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_init.py:_log_setup():429] Logging internal logs to /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/logs/debug-internal.log
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_init.py:init():462] calling init triggers
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {'workers': 5, 'print_freq': 1, 'checkpoint_dir': PosixPath('checkpoint'), 'vg_img_path': '/home/aneesh/datasets/gqa_imgs/images/', 'gqa_ann_path': '/home/aneesh/datasets/gqa_ann/OpenSource/', 'gqa_split_type': 'balanced', 'mask_model': 'none', 'masks': False, 'epochs': 25, 'imset': 'train', 'batch_size': 128, 'learning_rate': 0.2, 'dropout': 0.01, 'weight_decay': 1e-06, 'momentum': 0.9, 'clip': 1, 'betas': (0.9, 0.98), 'eps': 1e-09, 'loss_fn': 'cross_entropy', 'optimizer': 'adam', 'simg': 7, 'itersimg': 5, 'slotdimimg': 256, 'stext': 7, 'iterstext': 5, 'slotdimtext': 256, 'nhead': 4, 'tdim': 256, 'nlayers': 3, 'text_encoder_type': 'openai/clip-vit-base-patch32', 'rank': 0, 'dist_url': 'tcp://localhost:58472', 'world_size': 1, 'ngpus_per_node': 1}
2022-06-15 16:53:07,770 INFO    MainThread:225474 [wandb_init.py:init():515] starting backend
2022-06-15 16:53:07,772 INFO    MainThread:225474 [backend.py:ensure_launched():217] starting backend process...
2022-06-15 16:53:07,772 INFO    MainThread:225474 [backend.py:ensure_launched():222] started backend process with pid: 0
2022-06-15 16:53:07,772 INFO    BackendThr:225474 [internal.py:wandb_internal():90] W&B internal server running at pid: 225474, started at: 2022-06-15 16:53:07.772199
2022-06-15 16:53:07,772 INFO    MainThread:225474 [wandb_init.py:init():525] backend started and connected
2022-06-15 16:53:07,779 INFO    MainThread:225474 [wandb_init.py:init():596] updated telemetry
2022-06-15 16:53:07,780 INFO    WriterThread:225474 [datastore.py:open_for_write():75] open: /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/run-172yuf0n.wandb
2022-06-15 16:53:07,789 INFO    MainThread:225474 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-06-15 16:53:08,113 INFO    SenderThread:225474 [dir_watcher.py:__init__():197] watching files in: /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/files
2022-06-15 16:53:08,113 INFO    SenderThread:225474 [sender.py:_start_run_threads():811] run started: 172yuf0n with start time 1655304787
2022-06-15 16:53:08,115 INFO    MainThread:225474 [wandb_run.py:_on_init():1924] communicating current version
2022-06-15 16:53:08,115 INFO    SenderThread:225474 [sender.py:_save_file():948] saving file wandb-summary.json with policy end
2022-06-15 16:53:08,169 INFO    MainThread:225474 [wandb_run.py:_on_init():1928] got version response upgrade_message: "wandb version 0.12.18 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-06-15 16:53:08,169 INFO    MainThread:225474 [wandb_init.py:init():662] starting run threads in backend
2022-06-15 16:53:09,114 INFO    Thread-10 :225474 [dir_watcher.py:_on_file_created():244] file/dir created: /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/files/code/train.py
2022-06-15 16:53:09,115 INFO    Thread-10 :225474 [dir_watcher.py:_on_file_created():244] file/dir created: /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/files/wandb-summary.json
2022-06-15 16:53:09,115 INFO    Thread-10 :225474 [dir_watcher.py:_on_file_created():244] file/dir created: /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/files/diff.patch
2022-06-15 16:53:09,115 INFO    Thread-10 :225474 [dir_watcher.py:_on_file_created():244] file/dir created: /home/aneesh/github/slot_vqa/wandb/run-20220615_165307-172yuf0n/files/code
2022-06-15 16:53:12,455 WARNING MainThread:225474 [wandb_init.py:init():1020] interrupted
Traceback (most recent call last):
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 999, in init
    run = wi.init()
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 673, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 234, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 440, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "/home/aneesh/.conda/envs/slotvqa/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2022-06-15 16:53:12,460 WARNING MsgRouterThr:225474 [router.py:message_loop():73] EOFError seen in message_loop
2022-06-15 16:53:12,460 INFO    MainThread:225474 [internal.py:handle_exit():80] Internal process exited
